react-intervaler/src/App.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import './App.scss';
+import './stylesheets/index.scss';
+import Stopwatch from './components/Stopwatch'
+import About from './components/About'
+import Timer from './components/Timer'
+import Error from './components/Error';
+import Navigation from './components/Navigation';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+class App extends Component {
+  render() {
+    return (      
+       <BrowserRouter basename={process.env.PUBLIC_URL}>
+        <div>
```

Comment:
If you want to return multiple elements inside of a component that expects only one, you can use React.Fragment to do this.

```javascript
<React.Fragment>
   <Navigation />
            <Switch>
             <Route path="/" component={Stopwatch} exact/>
             <Route path="/timer" component={Timer} exact/>
             <Route path="/about" component={About} exact/>
             <Route component={Error}/>
           </Switch>
<React.Fragment>
```

The shorthand for fragments is just a tag like
```javascript
<>
...
</>
```

Reply:
Why do i need to or want to do this?

Reply:
It's just to prevent unnecessary nesting of useless divs. End of day, it's more of a convention rather than something that would affect debugging or code at runtime

Reply:
useful for when i need to pass a single item in, instead of using divs, not currently needed for this code however, just removed the div line completely